libqtile/resources/default_config.py

```diff
@@ -91,11 +99,11 @@
     ])
 
 layouts = [
+    layout.Columns(border_focus_stack='#d75f5f'),
```

Comment:
Why overriding the default color?

Reply:
because we have same color on unsplit column and simple window borders. I found it useful to set different colors to have some kind of indicator to see is it splitted or unsplit column.

Reply:
there was thoughts update floating window border, too. Because colors what we have in default config looks to different. That dark blue could be light blue (cyan) as we have current group border or Qtile logo.
P.S. maybe we should have Qtile logo in bar as image?

Reply:
Maybe we should keep the default here and do another branch for saner default colors across all layouts?

Reply:
will wait a little bit for more comments on other stuff (CurrentLayoutImage widget instead text) and can revert. I agree it is better to select default pallete of colors and set in all project defaults. @m-col suggested to have some theme.

Reply:
and for colors we need https://github.com/qtile/qtile/issues/1595 to start for this `themer`. Then we could use that themer to set defaults in widgets, layouts etc.